### DISCUSSION TITLE
Are we gonna get the feature that gives the channel info and permissions and everything else?

### DISCUSSION THREAD

**Discussion by Benjamin2722** [JustOptimize/return-ShowHiddenChannels#64] [Q&A] [answered]
For example, it used to show the bitrate, who can view/join, and other stuff about a vc.

same for regular channels, it would show what roles/members are allowed to view or not allowed to view a channel.

is this going to be added back?

**Comment by JustOptimize** [comment, marked as answer]
As you can see in the readme there's already a plugin to do that

**Reply by Benjamin2722** [reply]
Hi, that plug-in actually does something completely different from what I’m asking. That plug-in does that same thing as the Staff-Tag plug-in.

I'm asking if you’re going to add back the feature showhiddenchannels had where we could view all the info about a channel/vc and see the roles that can access/view that channel.

**Reply by JustOptimize** [reply]
For the
> info about the channel

I was thinking to bring back a similar page to the old one (The "Channel is Locked" page)

And for the
> see the roles that can access/view that channel.

The plugin I suggetsted actually allows you to do exactly that.
By right clicking the channel and then clicking on the permission context menu you can see all the permission that has been set up for that single channel allowing you to see what roles have the `View Channel` permission and what not

**Reply by Benjamin2722** [reply]
Ah, I see, thank you so much!
